README.md

```diff
@@ -206,6 +206,8 @@ Also note that we run some `clang-tidy` linting over the code as well (see `.cla
 
 `scripts/clang-tidy-only-diff.sh` is run in CI and will the build if it detects any issues.
 
+Additionally a check with address sanitizer is run in CI. It is recommended to perform testing with sanitizer locally prior to commiting. In order to compile with sanitizer use flag `-DENABLE_SANITIZER=ON` during `cmake` confiuration step. As long as leak sanitizer (which is a part of address sanitizer) is not currently supported across different platforms it is disabled in the CI. You can disable it loclally with the environment variable `ASAN_OPTIONS=detect_leaks=0`.
```

Comment:
spelling: 
- confiuration => configuration
- loclally.=> locally

Reply:
We should also note that ASAN should be run with debug symbols (`-DCMAKE_BUILD_TYPE=Debug`)?

Reply:
Done